app/src/common/shared/org/mozilla/vrbrowser/search/SearchEngineWrapper.java

```diff
@@ -102,13 +104,14 @@ private String getSuggestionURL(String aQuery) {
         return mSearchEngine.buildSuggestionsURL(aQuery);
     }
 
-    public void getSuggestions(String aQuery, SuggestionsDelegate delegate) {
+    public CompletableFuture<List<String>> getSuggestions(String aQuery) {
+        CompletableFuture<List<String>> future = new CompletableFuture<>();
         // TODO: Use mSuggestionsClient.getSuggestions when fixed in browser-search.
```

Comment:
Looks like this comment can be removed?

Reply:
Ideally we should use that method but I couldn't make it work using the current Kotlin-Java interop for calling suspend coroutines. I think it's ok to leave and I'll take a deeper look or maybe ask the AC team, they might know if there is a way.

Reply:
Okay, I saw `SuggestionsClient.getSuggestions() and assumed we were now calling what the `TODO` was highlighting.